t/22-svirt.t

```diff
@@ -21,6 +22,8 @@ BEGIN {
     unshift @INC, '..';
 }
 
+bmwqemu::init();
```

Comment:
I am ok with this change to make the tests look nice but without any kind of programmatic check I doubt it will hold very long. The next contributor to tests will forget to add this in any new module and also it's not very clear what `bmwqemu::init()` does here.